packages/react-components/package.json

```diff
@@ -476,19 +475,6 @@
     "./Upload": "./Upload.js",
     "./VerticalLayout": "./VerticalLayout.js",
     "./VirtualList": "./VirtualList.js",
-    "./css/Lumo.css": "./css/Lumo.css",
-    "./css/lumo/Badge.css": "./css/lumo/Badge.css",
-    "./css/lumo/Color.css": "./css/lumo/Color.css",
-    "./css/lumo/ColorBase.css": "./css/lumo/ColorBase.css",
-    "./css/lumo/Font.css": "./css/lumo/Font.css",
-    "./css/lumo/FontIcons.css": "./css/lumo/FontIcons.css",
-    "./css/lumo/Globals.css": "./css/lumo/Globals.css",
-    "./css/lumo/Sizing.css": "./css/lumo/Sizing.css",
-    "./css/lumo/Spacing.css": "./css/lumo/Spacing.css",
-    "./css/lumo/Style.css": "./css/lumo/Style.css",
-    "./css/lumo/Typography.css": "./css/lumo/Typography.css",
-    "./css/lumo/UserColors.css": "./css/lumo/UserColors.css",
-    "./css/lumo/Utility.module.css": "./css/lumo/Utility.module.css",
```

Comment:
While loading Lumo is otherwise a one-liner, this utility module could cause a lot of changes as using it would leave references all over the codebase. Let's discuss with the team how to proceed with it.

There also needs to be a note about removing the CSS files from this package in the upgrade guide.